### ISSUE TITLE
Indent code consistently

### ISSUE THREAD

**Issue by kocher** [solrmarc/solrmarc#38]
Code indentation is different (tabs, 2 spaces, 4 spaces, ...) all over the whole code base which makes applying changes cumbersome.

**Comment by haschart** [comment]
I agree that the code should be indented consistently, and it should be 4 spaces everywhere.   However this entire project is about to be supplanted by a completely re-written one, which currently is in a separate repo named solrmarc_the_next_generation.  Therefore efforts to make the code here easier to maintain or extend would be largely wasted.   

I appreciate your interest in this project and would welcome further interest and input for the new version.
